main/autogen/autoloader.cc

```diff
@@ -210,7 +224,17 @@ string DefTree::renderAutoloadSrc(const core::GlobalState &gs, const AutoloaderC
             predeclare(gs, fullName, buf);
         }
 
-        if (!children.empty()) {
+        if (pkgName.exists()) {
+            ENFORCE(!gs.packageDB().empty());
+
+            auto &pkg = gs.packageDB().getPackageInfo(pkgName);
+
+            // First path prefix is guaranteed to be the directory location of the package
+            const string_view pathPrefix = pkg.pathPrefixes()[0];
+
+            fmt::format_to(std::back_inserter(buf), "\n{}.pbal_register_package({}, '{}')\n", alCfg.registryModule,
+                           fullName, std::move(pathPrefix));
```

Comment:
you shouldn't need to `move` a `string_view` since it's a ptr-like type

Reply:
Ah whoops, I thought I had taken care of this. Will fix

Reply:
fixed